### DISCUSSION TITLE
Setting initial errors

### DISCUSSION THREAD

**Discussion by KyorCode** [react-hook-form/react-hook-form#3095] [Q&A]
**Situation**
After submitting my form, the formValues are validated on the server ( Client side validation is minimum due to security policy ). I then get a response from the server with the validation errors mapped in an object based on the form field names.

**Question**
Is there a way to set some sort of initialErrors on the useForm hook? Or an alternative to set the errors on the Form, without buliding additional logic with setError.

Thank you in advance.

**Comment by bluebill1049** [comment]
we don't have features like set initalErros.
